.github/workflows/packages-build.yml

```diff
@@ -0,0 +1,160 @@
+
+      - name: Prepare build environment.
+        shell: bash
+        run: |
+          python3 agent_build/build_package.py ${{ matrix.package_type }} prepare-environment --cache-dependencies-path ~/dependencies_cache
+
+          # in some cases, the 'prepare-environment' command also generates file 'paths.txt'. This file contains paths
+          # to binaries of all tools that are needed for the build. Use this file to add those paths to the PATH env.
+          # variable to be able to use those tools.
+          if [ -f ~/dependencies_cache/paths.txt ]; then
+            cat ~/dependencies_cache/paths.txt
+            cat ~/dependencies_cache/paths.txt >> $GITHUB_PATH
+          fi
+
+          # Remove checksum file
+          rm checksum.txt
+
+
+
+      - name: Build.
+        shell: bash
+        run: |
+          echo "eqwqwqww"
+          echo "$Env:Path"
+          echo "555555"
+          echo "$PATH"
+          python3 agent_build/build_package.py ${{ matrix.package_type }} build --output-dir agent-build-output
+          ls -al agent-build-output/${{ matrix.package_type }}
+          ls -al agent-build-output/${{ matrix.package_type }}/frozen_binary_test/
+          #mv agent-build-output/${{ matrix.package_type }}/frozen_binary_test/test_scalyr-agent agent-build-output/${{ matrix.package_type }}/frozen_binary_test/test_scalyr-agent${{ matrix.package_type }}
+
+
+
+
+
+#      - name: Save the package.
+#        uses: actions/upload-artifact@v2
+#        with:
+#          name: "package-${{ matrix.package_type }}"
+#          path: |
+#            agent-build-output/package/*.deb
+
+
+      - name: Save the package.
+        uses: actions/upload-artifact@v2
+        with:
+          name: "package-${{ matrix.package_type }}"
+          path: |
+            agent-build-output/${{ matrix.package_type }}/*.${{ matrix.package_type }}
+
+
+
+      - name: Save the frozen test script.
+        uses: actions/upload-artifact@v2
+        with:
+          name: "package-test-executor-${{ matrix.package_type }}"
+          path: |
+            agent-build-output/${{ matrix.package_type }}/frozen_binary_test/test_scalyr-agent*
+
+
+
+
+  test:
+    needs: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-20.04"]
+        docker-image: ["ubuntu:14.04", "ubuntu:16.04", "ubuntu:18.04", "ubuntu:20.04", "centos:7", "centos:8", "amazonlinux:2"]
+        package_type: ["deb"]
+        python-version: [3.8]
+        include:
+          - os: windows-2019
+            package_type: "msi"
+            python-version: "3.8"
+#          - os: ubuntu-20.04
+#            docker-image:
+#            package_type: "msi"
+#            python-version: "3.8"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Create requirements file
+        run: |
+            echo  "pytest" > pytest_requirments.txt
```

Comment:
Should we pin it to a specific version? Should we use requirements file?

Reply:
For now, I removed the pytest dependency for the package test at all.